pyproject.toml

```diff
@@ -12,7 +12,8 @@ readme = "README.md"
 version = "0.11.0"
 
 [tool.poetry.dependencies]
-certifi = "^2021.10.8"
+# For certifi, use ">=" instead of "^" since it upgrades its "major version" every year
+certifi = ">=2022.12.07"
```

Comment:
I think we want `>=2023.7.22`: https://github.com/groundlight/python-sdk/security/dependabot/12

Reply:
Thanks.  I'll update.